src/podman.scss

```diff
@@ -145,6 +145,27 @@
     padding-left: 30px;
 }
 
+.vertical-group {
```

Comment:
Same here - these are only be used on imageSearchModal.jsx file. I really prefer per-component CSS overrides and only in the case where we have global fixes above PF we should bring it up to files that are imported from the base of the package.

Reply:
Yep. Fully agreed.

Reply:
Also @marusak. But I should've removed this CSS when I removed vertical-group from the layout (and just used a ct-form-split class instead). Doing so now.

Reply:
I removed it. I am squashing those commits + breaks one test so fixing that. I'll push force once I am done.